ch_tools/monrun_checks/ch_orphaned_objects.py

```diff
@@ -4,12 +4,24 @@
 
 from ch_tools.chadmin.cli.object_storage_group import (
     ORPHANED_OBJECTS_SIZE_FIELD,
-    STORE_STATE_PATH,
+    STORE_STATE_LOCAL_PATH,
 )
+from ch_tools.chadmin.internal.zookeeper import check_zk_node, get_zk_node
 from ch_tools.common.result import CRIT, OK, WARNING, Result
 
 
 @click.command("orphaned-objects")
+@click.option(
+    "--store-state-local",
+    "store_state_local",
+    is_flag=True,
+    help="Get total size of orphaned objects from local file.",
+)
+@click.option(
+    "--store-state-zk-path",
+    "store_state_zk_path",
+    help="Zookeeper node path from which the total size of orphaned objects will be taken.",
+)
```

Comment:
It looks a bit strange that these options start with `--store` as the command only reads the state. Suggest to rename the options to `--state-local` and `--state-zk-path` (or to something else without `--store` suffix).

Reply:
thank, done